Android/app/src/main/java/com/example/androidgui/manageractivities/SelectEditOrDeleteActivity.java

```diff
@@ -48,12 +47,16 @@ protected void onCreate(Bundle savedInstanceState) {
         selectEditOrDelete.setDisplayedValues(selectOption);
     }
 
+    /**
+     * Manager selects manage menu.
+     *
+     * @param v android view.
+     */
     @RequiresApi(api = Build.VERSION_CODES.KITKAT)
     public void manageMenu(View v){
         String action = selectOption[selectEditOrDelete.getValue()];
         Intent extras = getIntent();
         String dishName = extras.getStringExtra("dishSelected");
```

Comment:
might want to consider using enums string value when getting extra, especially if it's used more than once